Automattic/jetpack
# Connection: Fix access to display_name property in connection status card

Fixes lack of display name when showing the "Logged in as ... " message.

#### Changes proposed in this Pull Request:

* Fixes property traversing for user object


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* On a connected site with the Backup plugin installed.
* Checkout this branch and build backup `jetpack build plugins/backup`
* Visit the backup page.
* Confirm you see your username on the Connection Status Card.


#### After 

![image](https://user-images.githubusercontent.com/746152/151016022-98b525d5-af18-4581-acc9-ba34ae81d9f3.png)

#### Before

![image](https://user-images.githubusercontent.com/746152/151016203-472ffa97-6dff-4fdf-be6e-270a643f994c.png)
